assignments/lambda-classes.js

```diff
@@ -129,26 +129,54 @@ class Instructor extends Person {
         const points = Math.floor(Math.random() * 101) 
 
         // -1 or 0 
-        const posOrNeg =  Math.floor((Math.random() * 2) * -1); 
-
+        const posOrNeg = (Math.floor(Math.random() * 2)) * -1; 
```

Comment:
Even when you write more complicated functions, it is easy to follow along because your code is well written.